### PR TITLE
Elaborate on file format support

### LINKED ISSUE #135: Add content to the File format chapter (7.4)
**Is your feature request related to a problem? Please describe.**
(TBD) Add general information and references for the 3 file formats glTF, FBX and USD.

**Describe the solution you'd like**
Add general information. Potential sources of information are

glTF
* https://www.khronos.org/Gltf
* https://en.wikipedia.org/wiki/GlTF

FBX
* https://www.autodesk.com/products/fbx/overview
* https://en.wikipedia.org/wiki/FBX

USD
* https://openusd.org/release/index.html
* https://de.wikipedia.org/wiki/Universal_Scene_Description

**Describe alternatives you've considered**
-

**Additional context**
-

### PR DESCRIPTION
**Pull request by ClemensLinnhoff**
## Describe your changes

Add links to 3D model file format specifications and state special considerations.

## Issue ticket number and link

Fixes #135 

## Checklist before requesting a review

- [x] I have performed a self-review of my code/documentation.
- [x] My changes generate no new warnings during the documentation generation.

### CODE REVIEW

File: content/geometry/file-format-support.adoc [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
-* FBX
-* USD
+ASAM OpenMATERIAL supports the 3D model file formats glTF, FBX and USD with the following considerations.
+
+**glTF**
+
+ASAM OpenMATERIAL 3D models in glTF format SHALL conform to the official https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html[glTF specification].
+
+The glTF coordinate system is defined as a right-handed system with +Y as up, +Z as forward, and -X as right. The front of a glTF asset faces +Z.
+This differs from how coordinate systems are defined in OpenMATERIAL, see xref:general.adoc[].
```
Comment (ClemensLinnhoff) [inline content/geometry/file-format-support.adoc:20]: @FKlopfer could you add a reference here to the "Coordinate systems" section of the general file? I was not sure how to reference a certain section within a file.

**Review by AsamDiegoSanchez — APPROVED** [review]
Reviewed
Signed-Off by: Diego Sánchez <diego.sanchez@asam.net>

**Review by ipg-sig — APPROVED** [review]

**Review by lyndyRott — APPROVED** [review]

File: content/geometry/file-format-support.adoc (hunk 1 above)
Comment (FKlopfer) [reply content/geometry/file-format-support.adoc]: To reference a subsection, you have to add the automatically created anchor to the link. In this case, it is like this: "xref:geometry/general.adoc#_coordinate_systems[Coordinate sytems]"

**Review by FKlopfer — APPROVED** [review]

**Review by LudwigFriedmann — APPROVED** [review]
Looks good to me!

**Pull request merged**